src/st/e_particles_maria1.h

```diff
@@ -11,8 +11,8 @@ void func_psp_0923AD68(Entity* self) {
     u16 direction;
     Entity* pl;
 
-    if (D_psp_091CF3DC && D_psp_091CF3A0) {
-        pl = D_psp_091CF3A0;
+    if (g_Player.unk5C && g_Player.unk20) {
+        pl = (Entity*)g_Player.unk20;
```

Comment:
Possible this is just an `Entity*` on PSP, but elsewhere on PSX/HD it is used as an s32. Need more PSP decompiled to know for sure.